state/protocol/badger/mutator_test.go

```diff
@@ -1701,6 +1700,9 @@ func TestEpochFallbackMode(t *testing.T) {
 			require.NoError(t, err)
 			safetyThreshold := rootSnapshot.Params().EpochCommitSafetyThreshold()
 			require.NoError(t, err)
+			rootProtocolState, err := rootSnapshot.ProtocolState()
+			require.NoError(t, err)
+			epochExtensionViewCount := rootProtocolState.GetEpochExtensionViewCount()
 
 			calculateExpectedStateId := calculateExpectedStateId(t, mutableProtocolState)
```

Comment:
this variable declaration currently shadows the function definition. We could rename the variable here to avoid shadowing:
```suggestion
			expectedStateIdCalculator := calculateExpectedStateId(t, mutableProtocolState)
```